TrafficCapture/trafficReplayer/src/test/java/org/opensearch/migrations/replay/FullTrafficReplayerTest.java

```diff
@@ -24,7 +25,7 @@ public void fullTest() throws Exception {
 
         try (var os = new NullOutputStream();
              var bos = new BufferedOutputStream(os);
-             var trafficSource = new V0_1TrafficCaptureSource("migrationLogs/kafkaOutput_09_23.proto.gz");
+             var trafficSource = new V0_1TrafficCaptureSource("/Users/schohn/migrationLogs/kafkaOutput_09_23.proto.gz");
```

Comment:
I'm guessing this was accidentally included. Can you remove this please?

Reply:
Thanks!